system-addon/test/schemas/pings.js

```diff
@@ -31,6 +31,9 @@ const UserEventAction = Joi.object().keys({
       "SEARCH",
       "BLOCK",
       "DELETE",
+      "DELETE_CONFIRM",
```

Comment:
Was this supposed to be used somewhere?

Reply:
The implementation of the Dialog component changed from being specific to deleting history items to a generic component, those are events that would have made sense in the first implementation. They should be removed.